### PR TITLE
Add xdebug

### PR DESCRIPTION
**Pull request by meron1122**

### CODE REVIEW

**Comment by maxpou** [comment]
Hello @meron1122,
Thank you for you Pull Request.
I didn't use xdebug. So, I need someone else to confirm this PR...

File: README.md [hunk 1]
```diff
@@ -150,6 +150,10 @@ Run `docker-compose up -d` instead.
 
 * Permission problem? See [this doc (Setting up Permission)](http://symfony.com/doc/current/book/installation.html#checking-symfony-application-configuration-and-setup)
 
+* How to config Xdebug?
+Xdebug is configured out of the box!
+Just config your IDE to connect port  `9001` and id key `PHPSTORM`
```
Comment (cuewizchris) [inline README.md]: This needs more clarification.
Whenever I'm trying to enable PHPStorm's listening function, I get the error "Can't start listening for connections from 'xdebug': Port 9000 is busy", no matter what I try. Perhaps you would be so kind as to elaborate on what you mean by this?

Comment (meron1122) [inline README.md]: Yep, just set  here:
phpstorm-xdebug-config

Port 9001.

Comment (cuewizchris) [inline README.md]: Done that. Configured a server with path mappings etc.
Don't know what I'm missing. I'd love to give maxpou the feedback that this works so he can accept the PR, but I'm too dumb to get it working. :D

Comment (cuewizchris) [inline README.md]: I knew I was missing something vital. Thank you so much, it works perfectly! :)

Comment (meron1122) [inline README.md]: Finaly! :) Regards!.

**Comment by cuewizchris** [comment]
@maxpou With a little help from @meron1122, I can now confirm that his additions work perfectly. I suggest you accept his PR. :+1:

**Comment by maxpou** [comment]
Good job and thank you guys :)

**Pull request merged**